compiler/ras/CallStack.cpp

```diff
@@ -168,8 +172,15 @@ const char *TR_PPCCallStackIterator::getProcedureName()
     strncpy(z, x, name_len);
     z[name_len] = '\0';
     char *ptoc;
-    Name *n = Demangle(z, ptoc); // Probably does a new...which is bad!
```

Comment:
It seems to me that the comment is still valid, the Name n is leaking. We should copy the text out and delete the name object. Or at least, don't remove the comment :)

From the docs[1]:
> When you are done with name, you must delete it.

And a comment from the demangle header:

> The most common operation is expected to be simply the retrieval of the
string representation of the demangled name, and is accomplished by the
virtual "Name" member "Text", which returns a pointer to a string. This
string is part of the Name object; if a user wants to change it or have
it exist beyond the lifetime of the object, s/he must make a copy of the
string. There is also a "char *" conversion operator for "Name" (and its
derived classes).